### ISSUE TITLE
Site is broken when viewed through github pages

### ISSUE THREAD

**Issue by jlndk** [travisneilson/DevTipsShowWebsite#7]
When i tried to view the site, through the link http://travisneilson.github.io/DevTipsShowWebsite, the website weren't styled properly.

It looked like this:
![screencapture-travisneilson-com-devtipsshowwebsite-1459336768299](https://cloud.githubusercontent.com/assets/3009101/14140590/3b113d50-f67a-11e5-9521-432640ecb9ed.png)

**Comment by andrealeone** [comment]
It is _**a matter of links**_:
you just need to remove the slash from the path to the main stylesheet for the homepage - for the other pages just add two dots at the root.

`<link href="assets/css/main.css" rel="stylesheet">` and
`<link href="../assets/css/main.css" rel="stylesheet">`

![link-fixing-one](https://cloud.githubusercontent.com/assets/16899766/14142145/1f362186-f684-11e5-87a9-61db7041239e.png)
![link-fixing-two](https://cloud.githubusercontent.com/assets/16899766/14142153/38571a9e-f684-11e5-9cb3-d12e8e2939ee.png)

It depends on Travis..

**Comment by andrealeone** [comment]
I'm going to "relativize" paths in [a separate branch](https://github.com/andrealeone/DevTips-ShowWebsite/tree/relativisation) to pull.

**Comment by travisneilson** [comment]
How big of an issue is this? I didn't even consider this use case because, why would anyone go there?

**Comment by andrealeone** [comment]
Yeah, you're right.. I did't think about it. --_It is still fun and a pleasure to try to fix bugs_.

**Comment by jlndk** [comment]
Since this website is properly going to be viewed by many of your viewers, i think it would be a good little fix, considering how little time it takes :).
